cogs/pug.py

```diff
@@ -1108,11 +1253,42 @@ async def adminremove(self, ctx, player: discord.Member):
         if self.pugInfo.removePlayerFromPug(player):
             await ctx.send('**{0}** was removed by an admin. {1}'.format(display_name(player), self.pugInfo.format_pug()))
 
+    @commands.command(aliases=['setserver','setactiveserver'])
+    @commands.has_permissions(manage_guild=True)
+    @commands.check(isPugInProgress_Warn)
+    async def adminsetserver(self, ctx, svindex: int):
+        """Overrides active server. Admin only"""
+        if self.pugInfo.gameServer.useServer(svindex):
+            await ctx.send('Server was activated by an admin - {0}.'.format(self.pugInfo.gameServer.format_current_serveralias))
+        else:
+            await ctx.send('Server **{0}** could not be activated.'.format(svindex))
+
+    @commands.command()
+    @commands.has_permissions(manage_guild=True)
+    @commands.check(isPugInProgress_Warn)
+    async def refreshservers(self, ctx):
+        """Adds a server to the available pool. Admin only"""
```

Comment:
This is displayed in the help, so should be accurate. Guessing this should be "Refreshes the available server pool. Admin only"?